annotate.py

```diff
@@ -203,8 +203,8 @@ def annotate(
             )
 
             if piece.name is not None and output_tsv_dir is not None:
-                piece_name = Path(piece.name.split(" ")[-1])
```

Comment:
Please edit this line to the following 2 lines:

```
# Removes the ID and first space from piece.name, and stems remaining path
piece_name = Path(" ".join(piece.name.split(" ")[1:])).stem
```

The idea is just to remove what's before the first space, and then stem. You're version would cause an error if there is no directory in piece.name.

Reply:
Did that in [7da775f](https://github.com/apmcleod/harmonic-inference/pull/73/commits/7da775f4d1e144d380b49b324273abbc11e1b527). However, when I last used annotate with the previous `piece_name = Path(piece.name).stem`, the output TSVs did not include any IDs.

> Your version would cause an error if there is no directory in piece.name.

I don't see this, what would be an example `piece.name` that would yield an error?

Reply:
I think for the ones you were testing, `piece.name` looked like `123: Bach/op12.msc`, where stem removes everything before the `/` and after the last `.`. I think it would error on something like `123: op12.msc`, although I'm not sure exactly what combination of args to `aggregate_corpus_data.py` would produce that (or if that's even possible, though I suspect it is).